combinations/hash.tsv

```diff
@@ -149,7 +149,7 @@ star=2.6.1d,samtools=1.10,gawk=5.1.0
 planemo=0.74.3,git=2.30.2,galaxyxml=0.4.14
 minimap2=2.18,samtools=1.12
 changeo=1.0.2,igblast=1.15.0,wget=1.20.1
-changeo=1.0.2,igblast=1.15.0,wget=1.20.1	quay.io/bioconda/base-glibc-busybox-bash:2.1.0	1
+changeo=1.3.0,igblast=1.20.0,wget=1.20.1	quay.io/bioconda/base-glibc-busybox-bash:2.1.0	1
```

Comment:
igblast 1.20 does not exist in bioconda https://anaconda.org/bioconda/igblast

Reply:
Thanks for spotting and sorry for the mistake. I've fixed it now 🤞